tests/Backend/Mixed/SharingTest.php

```diff
@@ -1162,6 +1162,106 @@ function (array $column) {
         $this->assertCount(5, $workspaceTableData);
     }
 
+    /**
+     * @dataProvider sharingBackendData
+     * @throws ClientException
+     */
+    public function testOrganizationShareBucket($backend)
+    {
+        $this->initTestBuckets($backend);
```

Comment:
Tohle by mohlo být klidně v `setUp` metodě

Reply:
Niektore testy pouzivaju tu metodu `initTestBuckets ()` inac ze tam maju iny `dataProvider` alebo tam nemaju vobec ten `dataProvider` to mi pride trochu zbytocne nejak podmienkovat v `setUp`